Model/Quote/SaveOrderBeforeSalesModelQuoteObserver.php

```diff
@@ -78,4 +92,45 @@ public function execute(\Magento\Framework\Event\Observer $observer)
         return $this;
     }
 
+    /**
+     * @param $quote
+     *
+     * @return bool
+     */
+    private function isMyParcelMethod($quote) {
+        $myParcelMethods = array_keys(Carrier::getMethods());
+        $shippingMethod  = $quote->getShippingAddress()->getShippingMethod();
+        /**
+         * @todo explode and use second part
+         * $shippingMethod = ...
+         */
```

Comment:
Dit weghalen, halve als dit toch nodig is. Even testen of dit werkt

Reply:
Done!